addons/account/models/account_payment.py

```diff
@@ -285,36 +280,46 @@ def _onchange_currency(self):
         if journal:
             return {'value': {'journal_id': journal.id}}
 
-    @api.multi
-    def _compute_payment_amount(self, invoices=None, currency=None):
+    @api.model
+    def _compute_payment_amount(self, invoices, currency, journal, date):
         '''Compute the total amount for the payment wizard.
```

Comment:
The docstring here is obsolete : new inputs and existing inputs are no longer optional